### PR TITLE
[bitnami/mariadb] add default binlog expiry configuration of 30 days

### LINKED ISSUE #21886: [bitnami/mariadb] Binlog cleanup configuration is missing
### Name and Version

bitnami/mariadb

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Run the pod a few months

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

_No response_

### What do you see instead?

Binlog files are accumulating without getting cleared taking half the space of the whole DB

### Additional information

MySQL is using a default for `binlog_expire_logs_seconds` of 30 days while MariaDB seems to throw that value over board https://mariadb.com/kb/en/replication-and-binary-log-system-variables/#binlog_expire_logs_seconds

When checking the `mariadb-galera` chart I found that there is `expire_logs_days` set to 14 days.

MySQL is dropping the `expire_logs_days` so I would suggest to also switch to `binlog_expire_logs_seconds` and set it to at least 14 days, like in mariadb-galera, in the next mariadb chart release.

### PR DESCRIPTION
**Pull request by AlexW00**
### Description of the change

- the change adds an entry `binlog_expire_logs_seconds=2592000` to the `primary.configuration` and `secondary.configuration` properties of values.yaml file
- this change will clean up binary logs older than 30 days, which currently does not happen (as discussed in #21886)

### Benefits

- old binary log files will not fill up the storage anymore

### Possible drawbacks

- binary log files, which are older than 30 days and are still needed, could get deleted

### Applicable issues

- fixes #21886

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mariadb/Chart.yaml [hunk 1]
```diff
@@ -34,4 +34,4 @@ maintainers:
 name: mariadb
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mariadb
-version: 19.0.7
+version: 19.0.8
```
Comment (dgomezleon) [inline bitnami/mariadb/Chart.yaml:37]:
```suggestion
version: 19.1.0
```

**Review by dgomezleon — COMMENTED** [review]
LGTM. Just bump the minor version to mark it as a significant change.

**Comment by AlexW00** [comment]
> LGTM. Just bump the minor version to mark it as a significant change.

Thx. I applied your suggested changes.

**Review by dgomezleon — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**